src/tasks/installers/helm.ts

```diff
@@ -28,6 +29,33 @@ export class HelmTasks {
         title: 'Verify if helm is installed',
         task: () => { if (!commandExists.sync('helm')) { command.error('E_REQUISITE_NOT_FOUND') } }
       },
+      {
+        title: 'Check Helm Version',
+        task: async (ctx: any, task: any) => {
+          try {
+            const version = await this.getVersion()
+            if (version.startsWith('v3.')) {
+              ctx.isHelmV3 = true
+            }
+            task.title = await `${task.title}: Found ${version}`
+          } catch (error) {
+            command.error(`Unable to get helm version. ${error.message}`)
+          }
+        }
+      },
+      {
+        title: `Create Namespace (${flags.chenamespace})`,
+        task: async (_ctx: any, task: any) => {
+          const che = new CheHelper(flags)
+          const exist = await che.cheNamespaceExist(flags.chenamespace)
+          if (exist) {
+            task.title = `${task.title}...does already exists.`
```

Comment:
I'm not sure but maybe

```suggestion
            task.title = `${task.title}...does already exist.`
```